Sources/AriesFramework/storage/Repository.swift

```diff
@@ -56,6 +56,10 @@ public class Repository<T: BaseRecord & Codable> {
         try await wallet.session!.update(operation: .remove, category: T.type, name: record.id, value: Data(), tags: nil, expiryMs: nil)
     }
 
+    public func deleteById(_ id: String, type: String) async throws {
```

Comment:
Because `Repository` is a generic class, you don't need to give a type. Just use `T.type` for type.

Reply:
PR has been improved.